tests/integration/customizations/s3/test_plugin.py

```diff
@@ -518,7 +518,7 @@ def test_copy_large_file_signature_v4(self):
             for i in range(num_mb):
                 f.write(b'a' * 1024 * 1024)
 
-        p = aws('s3 cp %s s3://%s/ --region eu-central-1' % (
+        p = aws('s3 cp %s s3://%s/ --region eu-central-1 --debug' % (
```

Comment:
That is just remnants from when I was trying to catch debug logs for a test that was failing. I think it was related to the sigv4 client signing.